ssas/service/public/router.go

```diff
@@ -35,6 +35,7 @@ func routes() *chi.Mux {
 	router := chi.NewRouter()
 	router.Use(service.NewAPILogger(), service.ConnectionClose)
 	router.Get("/token", token)
+	router.Post("/authn", VerifyPassword)
 	router.Post("/authn/request", RequestMultifactorChallenge)
```

Comment:
I think `request` is too generic for an endpoint. How about challenge? Then the sequence of requests would be:

POST /authn
POST /authn/challenge
POST /authn/verify
POST /authn/register